docs/reference/hacl/dh/ec/curve25519.md

```diff
@@ -0,0 +1,92 @@
+# Curve25519
+
+## API Reference
+
+`````{tabs}
+````{group-tab} 51
+```C
+#include "Hacl_Curve25519_51.h"
+```
+````
+````{group-tab} 64
```

Comment:
This is the one that only works on x64 systems (with assembly code and BMI2 & ADX on the CPU).